src/scenario.jl

```diff
@@ -776,3 +783,21 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
     site_ranks[site_ranks.==0.0] .= n_locs + 1
     return (raw=C_cover, seed_log=Yseed, fog_log=Yfog, shade_log=Yshade, site_ranks=site_ranks, bleaching_mortality=bleaching_mort, coral_dhw_log=collated_dhw_tol_log)
 end
+
+function _cyclone_mortality(coral_cover, coral_params, cyclone_mortality)::Nothing
```

Comment:
I don't think this is a private function as such.

The entire `run_model()` needs a clean up, but the idea would be to separate out each event into their own function and run in order of their envisioned sequence 

For example:

```
shade_locations!()
# ... etc...

bleaching_mortality!()
cyclone_mortality!()
# ... etc...

# run ODE
# ...
```

Reply:
This function modifies in place so by convention it should have the exclamation mark

```suggestion
function _cyclone_mortality!(coral_cover, coral_params, cyclone_mortality)::Nothing
```